src/main/webapp/scripts/classes/comment-class.js

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Class constructs Comment object from text string
+ *  responds with comment object that is ready for Perspective API use or error
+ */
+
+class Comment{
+  #MAX_COMMENTS = 3000;
+
+  constructor(text) {
+    this.text = text;
+    if(!this.text) {
+      console.error("User did not input any comment");
+      throw NO_COMMENT_ERROR;
+    }
+    if(this.text.length > this.#MAX_COMMENTS) {
+      console.error("User exceeded maximum comment length");
+      throw COMMENT_LENGTH_EXCEEDED_ERROR;
+    }
+    this.perspectiveString = this.makePerspectiveRequestString();
```

Comment:
I find `perspectiveString` to not be a property a comment. I would expect more to see a toxicity property. So maybe something like this.toxicity = <perspective_api_js_file>.getToxicity(this.text). This getToxicity() method knows about the perspective request body because its in the perspective api file. comment class file shouldn't know details about the perspective api. 

After this change, now you have a nice comment object with properties that you care about and in the `view` file from `MVC`  you know to use this comment object (displaying that this comment text has this comment toxicity).